tests/app/test_jinja_filters.py

```diff
@@ -10,6 +12,37 @@
 
 class TestJinjaFilters(TestCase):
 
+    def test_format_currency(self):
+        # Given
+        currency = 1.12
+
+        # When
+        format_value = format_currency(currency)
+
+        self.assertEquals(format_value, '£1.12')
+
+    def test_new_line_to_break(self):
+        # Given
+        new_line = 'this is on a\nnew line'
+        context = Mock()
+        context.autoescape = False
+
+        # When
+        format_value = nl2br(context, new_line)
+
+        self.assertEquals(format_value, '<p>this is on a<br>\nnew line</p>')
```

Comment:
I don't think it has any impact but is confusing so will change the implementation